### PR TITLE
Support ERB in YML

### PR DESCRIPTION
**Pull request by andydust**
So things like the ENV can be evaluated in them. Also works for _local files.

### CODE REVIEW

**Comment by andydust** [comment]
I cherry-picked the commits from the other PR over.

File: lib/lite_config.rb [hunk 1]
```diff
@@ -62,13 +67,19 @@ def config_path
...
+    name << '.erb' if erb?(name)
+    name
   end
 
   def local_config_filename(name)
     config_filename(name).gsub(/.yml$/, '_local.yml')
   end
 
+  def erb?(name)
+    File.exist?(name + '.erb') || File.extname(name) == '.erb'
```
Comment (federicokeen) [inline lib/lite_config.rb]: Is there any scenario where the extension of the name would be `.erb`? Usually name is just like `:aws_credentials`

**Comment by federicokeen** [comment]
Other than my comment looks good to me (well and travis being dumb)

File: lib/lite_config.rb (hunk 1 above)
Comment (andydust) [inline lib/lite_config.rb]: I'll rename this argument to filename, and also the local var in `#config_filename` - to avoid confusion. Thx.

**Comment by gtd** [comment]
:+1: Thanks y'all!

**Pull request merged**